style.css

```diff
@@ -0,0 +1,611 @@
+  position: absolute;
+  cursor: pointer;
+  width: 25px;
+  height: 25px;
+  top: 40px;
+  left: 310px;
+}
+
+.mobile-menu-list a {
+  color: #fff;
+  font-size: 30px;
+}
+
+.menu {
+  display: none;
+}
+
+.open {
+  font-size: 40px;
+  padding: 30px;
+}
+
+.body {
+  background-color: rgba(211, 211, 211, 0.4);
+}
+
+.main-text {
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-bottom: 80px;
+}
+
+.intro {
+  padding-top: 30px;
+  font-size: 30px;
+  color: #ec5242;
+}
+
+.title {
+  font-size: 40px;
+  color: #ec5242;
+  padding-top: 10px;
+}
+
+.information {
+  font-size: 16px;
+  width: 100%;
+  margin-top: 25px;
+  padding: 20px;
+  border: #fff solid 2px;
+}
+
+/* -----------------------main program-------------- */
+
+.program-container {
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  background-image: url(./images/background-pattern.png);
+}
+
+.main-program {
+  color: #fff;
+}
+
+.line {
+  background: #ec5242;
+  width: 90px;
+  height: 1px;
+  margin: 10px;
+}
+
+.method-container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+}
+
+.method {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(255, 255, 255, 0.1);
+  padding: 10px 20px;
+  gap: 15px;
+  color: #fff;
+}
+
+.method-icon {
+  font-size: 30px;
+}
+
+h3 {
+  color: #ec5242;
+}
+
+.method-info {
+  font-size: 12px;
+}
+
+.button-join {
+  color: #fff;
+  background: #ec5242;
+  padding: 15px 80px;
+}
+
+.button-desktop {
+  display: none;
+}
+
+/* -------------------------- */
+
+/* ------------------------about page---------------- */
+
+.about-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding-bottom: 30px;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+.about-information {
+  font-size: 16px;
+  margin-top: 50px;
+  padding: 30px 5px;
+  background: #fff;
+  border: #d3d3d3 solid 1px;
+}
+
+.about-info-mobile {
+  display: block;
+}
+
+.about-info-desktop {
+  display: none;
+}
+
+.contact-info {
+  font-size: 15px;
+  color: #272a31;
+  margin-top: 15px;
+}
+
+.email {
+  text-decoration: underline;
+  font-size: 15px;
+  color: #272a31;
+  margin-top: 5px;
+}
+
+/* --------------------------teachers-------------------- */
+
+.teachers {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 30px;
+  padding: 20px;
+}
+
+.teachers-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 30px;
+  padding: 20px;
+}
+
+.teachers-sample {
+  display: flex;
+}
+
+.teachers-images {
+  width: 150px;
+  height: 150px;
+}
+
+.short-line {
+  width: 40px;
+  height: 1px;
+  background: #272a31;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.teachers-info {
+  width: 100%;
+  float: right;
+}
+
+.teachers-name {
+  color: #272a31;
+}
+
+.teachers-company {
+  color: #ec5242;
+}
+
+.teachers-subject {
+  font-size: 14px;
+}
```

Comment:
- [ Optional ]  As is the case in the original template, kindly make sure when the page loads the user can only see two speakers on the page. and the SHOW MORE button. once clicked they can be able to see the all speaker's lists.

Your design 👇

![Capture](https://user-images.githubusercontent.com/93598189/211547092-35f8b9a5-2687-4eef-832e-06e1967934a6.PNG)

Original design 👇

![Capture](https://user-images.githubusercontent.com/93598189/211547345-8ea6a44e-8412-4c0c-b93d-94223cac6c6a.PNG)